protocol.md

```diff
@@ -395,8 +395,15 @@ Minimum Authentication Level: *Unknown*
 
 ### Emoji
 
-*TODO*
+The emoji system is still very unknown, but it seems that they are pulled from an array or table
+It seems that at one point it was previously located somewhere else, since the first returns a 301 and the second responds fine.
+
+
+[https://stream.mattkc.com/chat/emotes/]
 
+![ Status | Method | Domain | File | Initiator | Type | Transfered
+GET | (lock icon) stream.mattkc.com | emotes | jquery-3.6.1.min.js | html | 1.68 kB (raced)
+GET | (lock icon) stream.mattkc.com | /chat/emotes/ | jquery-3.6.1.min.js | html | 1.65 kB (raced)](images/protocolemoji.png)
```

Comment:
No idea what this is supposed to be.

Reply:
Protocol redirect connections

Reply:
> No idea what this is supposed to be.

For the original two images, you had quoted them in plaintext, like something for accessibility, so that's just what I assumed and wrote all of the things in the image out.

Reply:
I don't think we need to use captured web traffic in our documentation. I think it would be best if we told developers how to read the contents of the link and maybe have a table of emoji that we know of. Don't get me wrong though, I'm all for accessibility and I approve of making things easier to access. However, I want to make sure that the data at the link you provided is accessible to those reading the documentation, even if the contents of the link change.